cos-fleet-manager-api/src/openapi/specs/connector_mgmt.yaml

```diff
@@ -1309,7 +1309,7 @@ components:
                 name: "fieldName"
                 value: "fieldValue"
         desired_state: ready
-        error_handling:
```

Comment:
Epic Brief defines the field as `error_handler` not `error_handling`

Reply:
@lburgazzoli for what exactly? fixing the the example? I did that when I was working on this PR that introduces the central splitting and cleanup of the connector spec. It's already part of this PR.

Reply:
to fix the example in the cos-fleet-manager openapi (the connector_mgmt.yaml here is a copy of the fleet manager one)

Reply:
@lburgazzoli I filed https://issues.redhat.com/browse/MGDSTRM-7541